chall02/gsanchez.py

```diff
@@ -0,0 +1,56 @@
+# **************************************************************************** #
```

Comment:
Your code is missing the sheebang, therefore it won't be executed in python and will fail.
```suggestion
#!/usr/bin/env python3
# **************************************************************************** #
```